### PR TITLE
Text node updates and fixes

### PR DESCRIPTION
**Pull request by fakob**
* Added option which creates a rich text node if you paste html or plain text right into the canvas
  * Updated the way we are copying nodes -> now using the text/html mime type to better differentiate what is copy/pasted
* Added autoHeight option for the rich text node (on by default)
* Made the min-size of the rich text node smaller
* Moved the rest of the hybrid node functions into the hybridNode class

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -374,19 +373,63 @@ Viewport position (scale): ${viewportScreenX}, ${Math.round(
...
-      try {
-        const json = JSON.parse(textFromClipboard) as SerializedSelection;
-        if (json.version) {
-          e.preventDefault();
-          const pastedNodes = await currentGraph.current.pasteNodes(json, true);
-          console.log(pastedNodes);
-        } else {
-          console.error('Clipboard does not contain valid node data');
+      if (!isEventComingFromWithinTextInput(e)) {
+        const clipboardBlobs = await getDataFromClipboard();
```
Comment (magnificus) [inline src/App.tsx:377]: these two statements share a ton of code and should be consolidated into common function avoid copy pasted code

Comment (fakob) [reply src/App.tsx:377]: You are right, it looks like there is a lot of copy pasted code, but these 4 options are all slightly different. I have now created a common function, but it is not as small as you would have hoped I guess.

File: src/nodes/textEditor/textEditor.tsx [hunk 2]
```diff
@@ -245,6 +291,59 @@ export class TextEditor extends HybridNode {
...
+          const editorHeight = Math.ceil(
+            document.getElementById(this.id).getBoundingClientRect().height /
+              PPGraph.currentGraph.viewport.scale.x
+          );
+          this.resizeNode(this.nodeWidth, editorHeight);
+        }
+      };
+
+      // workaround to get ref of editor to be used as mounted/ready check
+      useEffect(() => {
```
Comment (magnificus) [inline src/nodes/textEditor/textEditor.tsx:305]: looks a lil bit sus

**Review by magnificus — APPROVED** [review]

File: src/nodes/textEditor/textEditor.tsx (hunk 2 above)
Comment (fakob) [reply src/nodes/textEditor/textEditor.tsx:305]: This is what slatejs people have suggested as there is nothing native built in (yet).
https://github.com/ianstormtaylor/slate/issues/3618#issuecomment-653635644

**Pull request merged**